docs/developer-docs/multi-chain/bitcoin/overview.mdx

```diff
@@ -9,6 +9,14 @@ import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
 
 <MarkdownChipRow labels={["Advanced", "Bitcoin" ]} />
 
+# Why it is important
+
+The integration of Internet Computer (ICP) with the Bitcoin network allows ICP smart contracts to directly interact with the Bitcoin network. 
+The is important because it brings smart contract functionality to Bitcoin, opening up new possibilities for DeFi and dapp development.
+For example, ICP smart contracts can directly hold, receive, and send BTC, and also facilitates the creation of a ckBTC token that is backed 1:1 by BTC.
```

Comment:
I think there is a grammar problem here. "And also facilitates" is singular, but the subject in the preceding sentence is in plural. I am not actually sure what the intention was.

Reply:
fixed